example_interfaces/CMakeLists.txt

```diff
@@ -11,9 +11,7 @@ find_package(builtin_interfaces REQUIRED)
 find_package(rosidl_default_generators REQUIRED)
 
 rosidl_generate_interfaces(${PROJECT_NAME}
-  "msg/LargeFixed.msg"
   "srv/AddTwoInts.srv"
-  DEPENDENCIES builtin_interfaces
```

Comment:
I `builtin_interfaces` is not used anymore anywhere it should also be removed from the manifest.